### ISSUE TITLE
Custom TTL support

### ISSUE THREAD

**Issue by rafeequl** [code-mancers/dockup#170]
if possible we could provide users with custom TTL of an deployment, sometimes we need more 24 hours for a particular branch, for example : Mobile API for Mobile team reference #

**Comment by iffyuva** [comment]
Yeah, this makes sense, lets work on it!

**Comment by emilsoman** [comment]
I agree there is a solid use case for this. Let's support this along with k8s refactor.
